src/columnsTypes/types/Label/utils.js

```diff
@@ -145,7 +145,13 @@ export const displayPopup = (view, popupDOM) => {
 export const calculateMenuPosition = (menuDOM, {node, dom: cellDOM, pos}) => {
   const {style} = menuDOM;
 
-  const {left, top, height: cellHeight} = cellDOM.getBoundingClientRect();
+  const {
+    left,
+    top,
+    height: cellHeight,
+    width: cellWidth,
+  } = cellDOM.closest('td').getBoundingClientRect();
```

Comment:
Will look way more simple after we will change the popup to be located by the + button